Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -305,3 +286,57 @@ you can export the settings from this cluster with the following steps.
 
 !!! important
     For other clusters to connect to storage in this cluster, Rook must be configured with a networking configuration that is accessible from other clusters. Most commonly this is done by enabling host networking in the CephCluster CR so the Ceph daemons will be addressable by their host IPs.
+
+## Upgrades
+
+Upgrading the cluster would be different for restricted caps and non-restricted caps,
+
+1. If consumer cluster doesn't have restricted caps, this will upgrade all the default CSI users (non-restricted)
+
+    ```console
+    python3 create-external-cluster-resources.py --upgrade
+    ```
+
+2. If the consumer cluster has restricted caps
+
+    Restricted users created using `--restricted-auth-permission` flag need to pass mandatory flags: '`--rbd-data-pool-name`(if it is a rbd user), `--k8s-cluster-name` and `--run-as-user`' flags while upgrading, in case of cephfs users if you have passed `--cephfs-filesystem-name` flag while creating CSI users then while upgrading it will be mandatory too. In this example the user would be `client.csi-rbd-node-rookstorage-replicapool` (following the pattern `csi-user-clusterName-poolName`)
+
+    ```console
+    python3 create-external-cluster-resources.py --upgrade --rbd-data-pool-name replicapool --k8s-cluster-name rookstorage --run-as-user client.csi-rbd-node-rookstorage-replicapool
+    ```
+
+    !!! note
+        1) An existing non-restricted user cannot be converted to a restricted user by upgrading.
+        2) The upgrade flag should only be used to append new permissions to users. It shouldn't be used for changing a CSI user already applied permissions. For example, be careful not to change pools(s) that a user has access to.
```

Comment:
I'm not 100% sure if this will render the way we want. Once this is merged, please check the master version docs to ensure it looks right.

Reply:
Yaa it is formated one after the another, looks fine, but I can shift it to the next line too
![Screenshot from 2024-07-26 12-00-17](https://github.com/user-attachments/assets/2116fd02-38a1-4765-bdb3-2bc91e7ef94e)